pkg/kernel/networkservice/connectioncontextkernel/ipcontext/iprule/common.go

```diff
@@ -254,6 +282,8 @@ func delRule(ctx context.Context, handle *netlink.Handle, policy *networkservice
 		return errors.WithStack(err)
 	}
 
+	// TODO: Flush table
```

Comment:
I also think that we need to delete an entry from Maps. Perhaps after deleting current NSC, a new one will come with the same `mechanism.GetNetNSURL()`. It may cause an error

Reply:
The point of having `mechanism.GetNetNSURL()` as key is to sync the table IDs for every connection handled by a pod, if we use, for instance, the connection ID as key, and the pod requests 2 NSM connections with policies, there might be table ID collision.

Otherwise, with the `mechanism.GetNetNSURL()` as key, I am not sure when to delete the entry from the map

Reply:
Yes, got it.
I thought that we can delete here an entry from `tableIDs.Load(mechanism.GetNetNSURL()).policies`
And somewhere here https://github.com/networkservicemesh/sdk-kernel/blob/main/pkg/kernel/networkservice/connectioncontextkernel/ipcontext/iprule/common.go#L246
we can check: 
```
if len(tableIDs.Load(mechanism.GetNetNSURL()).policies) == 0 {
  tableIDs.Delete(mechanism.GetNetNSURL()) 
  }
```
Do I understand correctly? Thoughts?

Reply:
Yes, I think this is a good idea. And probably we could iterate over the policy routes entries in the tableIDs map instead of the ones in the connection.

Also, before implementing it, flushing the table should be done since the counter will be reset. Otherwise when the entry will be remove and a new one will be added again, and tables will contain old routes.